.github/workflows/pr-close.yaml

```diff
@@ -0,0 +1,36 @@
+name: pr_close
+
+on:
+  pull_request:
+    branches: [ main ]
+    types: [closed]
+
+jobs:
+  job_id:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+
+    steps:
+
+    # Install gcloud, do not specify authentication.
+    - uses: 'google-github-actions/setup-gcloud@v0'
+      name: 'Set up Google Cloud SDK'
+      with:
+        project_id: 'kr-gcping'
+
+    # Configure Workload Identity Federation via a credentials file.
+    - id: 'auth'
+      name: 'Authenticate to Google Cloud using Workload Identity Federation'
+      uses: 'google-github-actions/auth@v0'
+      with:
+        workload_identity_provider: 'projects/793166746566/locations/global/workloadIdentityPools/github-pool/providers/my-provider'
```

Comment:
Super to see this using WIF

Reply:
Yep! Also moved the vars to use secrets instead.